internal/cmd/logs/logs.go

```diff
@@ -59,4 +61,7 @@ func init() {
 	Cmd.Flags().BoolVarP(&allLogs, "all", "a", false, "include uninteresting logs")
 	Cmd.Flags().BoolVarP(&chart, "chart", "c", false, "Output a gantt chart of the most recent action as an html file")
 	Cmd.Flags().BoolVar(&config.Debug, "debug", false, "Output debugging information")
+	Cmd.Flags().IntVarP(&logsLength, "length", "l", 0, "Number of logs to display")
+	Cmd.Flags().IntVarP(&logsDays, "days", "d", 0, "Age of the logs to display in days")
+	Cmd.MarkFlagsMutuallyExclusive("days", "length")
```

Comment:
Why are these exclusive? What if a single day has a large number of log entries?

Reply:
just did not test those things together, will add tests and remove that exclusive option, they should work like you said a test till solidify it.